.buildkite/scripts/steps/functional/e2e.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo "--- Trigger kibana e2e pipeline"
```

Comment:
Can we change the name here to `Trigger unsupported ftr tests`?

Reply:
Pushed [41bf6c0](https://github.com/elastic/kibana/pull/152846/commits/41bf6c0eef1f8ab82264375acefb5035fd430a5a)